packages/fuchsia_ctl/lib/src/amber_ctl.dart

```diff
@@ -59,10 +59,9 @@ class AmberCtl {
   Future<void> addPackage(String packageName) async {
     stdout.writeln('Adding $packageName...');
     final List<String> updateCommand = <String>[
-      'amberctl',
-      'get_up',
-      '-n',
-      packageName,
+      'pkgctl',
+      'resolve',
+      'fuchsia-pkg://fuchsia.com/$packageName',
```

Comment:
This will fail if the package server is started with a repo name that is not fuchsia.com. I don' think this is a concern right now since all the workflows work this way. but in the future we may need to update this method to take in the repo name as well.

Reply:
ack.